### ISSUE TITLE
[Bug] ClearButtonVisibility on Entry is currently not accessible

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#3384]
### Description

Clearbutton is just a drawable set on the underlying TextView via this
editText.SetCompoundDrawablesWithIntrinsicBounds(drawable, null, null, null)

So, it's not going to be reachable by a screen reader. This will mostly likely need to be solved via Custom Actions or similar

**Comment by v-longmin** [comment]
@PureWeen Could you share your  repro project/solution, thanks?

**Comment by jinxinjuan** [comment]
Verified this issue with Visual Studio Enterprise 17.6.0 Preview 3.0. Still repro on a MAUI sample project.
[MauiApp10.zip](https://github.com/dotnet/maui/files/11228539/MauiApp10.zip)
